try:
    raise BaseException("ok")
except:
    print("nice")
